### ISSUE TITLE
13.4 Failed to read the 'localStorage' property from 'Window':

### ISSUE THREAD

**Issue by stan-qaz** [brave/browser-laptop#7350]
Edited to correct errors.

- Did you search for similar issues before submitting this one?

Yes, might be related to: https://github.com/brave/browser-laptop/issues/6866

- Describe the issue you encountered:

On the dashboard the traffic graph and interface statistics are not updating and several other graphs in pfSense (firewall software) don't work in Brave browser v13.4, they seem to work fine in other browsers I've tried **Edit:** ~~and they were working in earlier versions of Brave.~~ The new firewall version isn't working in older Brave versions.

**Edit:** There was an update to the pfSense firewall that I was unaware of that appears to be the source of this issue, see my following comment for details.

- Platform Opensuse 42.2
**Edit** also Windows 10

- Brave Version (revision SHA):

Brave	0.13.4
rev	71d8ffc
Muon	2.56.4
libchromiumcontent	56.0.2924.87
V8	5.6.326.50
Node.js	7.4.0
Update Channel	dev
os.platform	linux
os.release	4.4.46-11-default
os.arch	x64

- Steps to reproduce:
    1. Open the home / dashboard web page of the pfSense firewall
    2. Wait for it to load and update the system status
    3. The traffic graph and interface statistics are never populated.
    
- Any related issues:

Viewing the dashboard I see these error messages on the console:

Dashboard:
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at http://172.16.0.1/:692:2
    at HTMLDocument.<anonymous> (http://172.16.0.1/js/pfSense.js:276:3)
    at i (http://172.16.0.1/vendor/jquery/jquery-1.12.0.min.js:2:27451)
    at Object.fireWith [as resolveWith] (http://172.16.0.1/vendor/jquery/jquery-1.12.0.min.js:2:28215)
    at Function.ready (http://172.16.0.1/vendor/jquery/jquery-1.12.0.min.js:2:30008)
    at HTMLDocument.K (http://172.16.0.1/vendor/jquery/jquery-1.12.0.min.js:2:30370)

Viewing status - monitoring  and selecting NTP gives this error message:

Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at calculate_summary (http://172.16.0.1/status_monitoring.php:1256:3)
    at Object.generate (http://172.16.0.1/status_monitoring.php:1242:5)
    at renderLoop (http://172.16.0.1/vendor/nvd3/nv.d3.js:99:27)

**Comment by bsclifton** [comment]
@stan-qaz would you be able to help debug a bit more?  Can you try the same site with shields off? Also, I'm curious if you had fingerprint protection on?  Please let us know 😄  Thanks!

**Comment by stan-qaz** [comment]
Shields are off, fingerprint protection is off.

I tried the firewall with an older Brave version, one that I know was working to view it (11.1) and it too is now failing to show the widgets so there was some change to the firewall hat I wasn't aware of that triggered this. 

Checking the firewall I see that it has been updated to version 2.3.3 from 2.3.1-1 with many changes to the GUI code and a change to nginx from lighttpd as the server.

https://doc.pfsense.org/index.php/2.3_New_Features_and_Changes#Dashboard.2FWidgets.2FGUI

I also borrowed a Windows 10 system, installed Brave 13.4 and the symptoms and error messages are the same on that, hopefully that will make testing easier for you.

Any other information I can provide or testing I can try?
